core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/components/dot-edit-content-sidebar-activities/dot-edit-content-sidebar-activities.component.html

```diff
@@ -1,30 +1,93 @@
+                            <div class="activities__item" data-testid="activity-item">
+                                <div class="activities__header">
+                                    <p-avatar
+                                        dotGravatar
+                                        [email]="activity?.email"
+                                        styleClass="activities__avatar" />
+                                    <span class="activities__user" data-testid="activity-user">
+                                        {{ activity.postedBy }}
+                                    </span>
+                                    <span class="activities__date" data-testid="activity-date">
+                                        {{ activity.createdDate | dotRelativeDate }}
+                                    </span>
+                                </div>
+                                <span
+                                    class="activities__description"
+                                    data-testid="activity-description">
+                                    {{ activity.commentDescription }}
+                                </span>
+                            </div>
+                        </div>
+                    }
+                </ng-template>
+                <ng-template pTemplate="empty">
+                    <span class="activities__empty" data-testid="empty-state">
+                        {{ 'edit.content.sidebar.activities.empty' | dm }}
                     </span>
-                    <span class="activities__date" data-testid="activity-date">
-                        {{ activity.createdDate | dotRelativeDate }}
-                    </span>
-                </div>
-                <span class="activities__description" data-testid="activity-description">
-                    {{ activity.commentDescription }}
-                </span>
-            </li>
-        } @empty {
-            <span class="activities__empty" data-testid="empty-state">
-                {{ 'edit.content.sidebar.activities.empty' | dm }}
-            </span>
+                </ng-template>
+            </p-dataView>
+        </div>
+        @if (!$hideForm()) {
+            <div class="activities__footer" data-testid="activities-footer">
+                <form
+                    [formGroup]="form"
+                    (ngSubmit)="onSubmit()"
+                    class="activities__form"
+                    data-testid="activities-form">
+                    <div class="activities__control">
+                        <textarea
+                            pInputTextarea
+                            [rows]="5"
+                            autoResize="false"
+                            formControlName="comment"
+                            data-testid="activities-input"
+                            class="activities__input"></textarea>
+
+                        <dot-field-validation-message
```

Comment:
Use self-closing components in this case: <dot-field-validation-message  [field]="form.get('comment')" />